Makefile

```diff
@@ -238,10 +238,15 @@ ossls-notice: deps
 ## Tests ##
 ###########
 
+.PHONY: test-prep
+test-prep:
+	@echo "+ $@"
+	@mkdir -p test-output
+
 .PHONY: unit-tests
-unit-tests: deps
+unit-tests: deps test-prep
 	@echo "+ $@"
-	go test -race ./...
+	go test -race -v ./... | tee test-output/test.log
```

Comment:
`-v` is required otherwise go-junit-report says there are 0 tests for each package